### ISSUE TITLE
Code is not here.

### ISSUE THREAD

**Issue by FibonacciDude** [EllaRahimian/FS-HGR#2]
Hello! The code for this paper is not in the repository yet, when will it be published? I really need it.

Thanks!

**Comment by FibonacciDude** [comment]
Of course, this is only an io page, but it would be nice if the actual code was here.

**Comment by sadathosseiny1367** [comment]
I couldn't find your code that you mentioned on your paper "FS-HGR: Few-shot Learning for Hand Gesture
Recognition via ElectroMyography".

**Comment by sadathosseiny1367** [comment]
I would appreciate it if you could provide your code here, as you claimed and mentioned in your paper.
